### ISSUE TITLE
Error enabling the Compute Engine API required to delete the default network

### ISSUE THREAD

**Issue by mifrank-cg** [terraform-google-modules/terraform-google-project-factory#813]
### TL;DR

I would like to create a project without a default vpc. 

But I am receiving the error:
Error: Error enabling the Compute Engine API required to delete the default network: failed to send enable services request: googleapi: Error 403: Service Usage API has not been used in project 533653607165 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/serviceusage.googleapis.com/overview?project=533653607165 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.

### Expected behavior

Project creation succeeded

### Observed behavior

The project creation fails and Terraform marks the project as "tainted", so it wants to recreate the project, but that is not possible, because the deletion just disables the project and the id is still existing for the next 30 days.

### Terraform Configuration

```hcl
module "project_sov_network" {
  source  = "terraform-google-modules/project-factory/google"
  version = "~> 14.1"

  name                           = "prj-${local.client}-sov-network"
  random_project_id              = true
  org_id                         = local.org_id
  billing_account                = local.billing_account
  enable_shared_vpc_host_project = true
  folder_id                      = data.google_active_folder.sov_client_folder.id
  labels = {
    client = local.client
  }
  activate_apis = [
    "compute.googleapis.com",
    "iam.googleapis.com",
    "iamcredentials.googleapis.com",
    "oslogin.googleapis.com",           // Required for compute.googleapis.com
    "servicenetworking.googleapis.com", // Required for private network setup
    "vpcaccess.googleapis.com",
    "container.googleapis.com",
  ]
}
```

### Terraform Version

```sh
Terraform v1.3.7
on windows_amd64
+ provider registry.terraform.io/gavinbunney/kubectl v1.14.0
+ provider registry.terraform.io/hashicorp/google v4.61.0
+ provider registry.terraform.io/hashicorp/google-beta v4.61.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.20.0
+ provider registry.terraform.io/hashicorp/null v3.2.1
+ provider registry.terraform.io/hashicorp/random v3.5.1
+ provider registry.terraform.io/hashicorp/time v0.9.1
```

### Additional information

I used the exact same Terraform configuration for another organization.
I just copied the code and changed the setting of the locals and variables.
According to Google the billing account and everything else is configured correctly.
To be honest, I am not sure, what else to check.

**Comment by mifrank-cg** [comment]
Just for your information: I am not using any service account or seed project for the creation.
I run the terraform codes manually.

**Comment by TheDoubleJo** [comment]
Any workaround ?

**Comment by mifrank-cg** [comment]
Unfortunately, I do not have a workaround. It sometimes happened. So, I ran the Terraform code until it did not happen again.
Regards

**Comment by ajaykumar-gbi** [comment]
Any solution for this?
